analysis/paper/shceof.Rmd

```diff
@@ -1128,7 +1147,7 @@ The reason the conventional EOF analysis arrives at the same separation than our
 To visualize that, Figure \@ref(fig:phase-histogram) shows an histogram that counts the number of SON years in which the cEOF2 was close to each of the four particular phases (positive/negative Real/Imaginary), with the observations for each season marked as rugs on the horizontal axis.
 For instance, years with cEOF2 phase within 45º of 0º are nearest the "positive Real" phase.
 About two thirds of time cEOF2 has a phase similar to either the negative or positive Imaginary phase.
-It could be seen that Imaginary phase is the most common phase and It is also the direction that has the maximum relationship with ENSO.
+It could be seen that Imaginary phase is the most common phase and It is also the direction that has the maximum relationship with ENSO. \textcolor{red}{Ojo que todavía no estuvo la discusión con lo del ENSO, así que esto lo deberías decir más adelante.}
```

Comment:
si, eso es cierto. Quizás entonces aclararía algo como "It is also **by construction** the direction that has the maximum relationship with ENSO"

Reply:
```suggestion
It could be seen that Imaginary phase is the most common phase and It is also, by construction, the direction that has the maximum relationship with ENSO. 
```